### ISSUE TITLE
Connecting to online CRM

### ISSUE THREAD

**Issue by janiskulj** [daryllabar/XrmUnitTest#53]
Does anyone else have problems connecting to Dynamics 365 Online?

I think the problem is reference to (old) Microsoft.Xrm.Client assembly. After some tweaking of connection settings and code I got stuck on "Access Denied" response. 

How should I connect to Dynamics 365 Online?

**Comment by daryllabar** [comment]
You've hit the nail exactly on the head.  I need to upgrade the client assemblies to use the new 9.0 assemblies because the TLS security has changed.  For now you'll be stuck in memory...  There is a lot of work that I'm still looking to do on this, but other projects are currently taking my time.

**Comment by janiskulj** [comment]
I did quick&dirty fix

1. Update all Microsoft.CrmSdk.* nuget packages from 8.* to latest version 
2. Added nuget package Microsoft.CrmSdk.XrmTooling.CoreAssembly (to all projects using 9.* version of Xrm dlls)
3. Changed method IClientSideOrganizationService CreateService(CrmServiceInfo info) of CrmServiceUtility class to:

```
            CrmServiceClient client = new CrmServiceClient(info.CrmServerUrl);
            return new ClientSideOrganizationService(client.OrganizationServiceProxy);
```
(reused CrmServerUrl setting for connection string)
